src/app/record/components/peer-review-stack/peer-review-stack.component.scss

```diff
@@ -0,0 +1,42 @@
+[body] {
+  display: flex;
+  flex-direction: column;
+}
+
+:host {
+  width: 100%;
+}
+
+:host.display-the-stack {
+  border-style: solid;
+  border-radius: 4px;
+  border-width: 2px;
+
+  app-panel:not(.last) {
+    display: block;
+    position: relative;
+  }
+}
+
+app-panel {
+  margin-top: 16px;
+  word-break: break-word;
+
+  p {
+    margin-top: 0;
+    margin-bottom: 5px;
+  }
+}
+
+.header-sub-peer-review {
+  margin-right: 40px;
```

Comment:
remember we need to use `margin-inline-end` so we support RTL

Reply:
Same for the other left/right margin and padding

Reply:
Done!